conf/hibench.conf

```diff
@@ -3,10 +3,10 @@
 hibench.scale.profile                 tiny
 
 # Mapper number in hadoop, partition number in Spark
-hibench.default.map.parallelism         8
+hibench.default.map.parallelism         138
```

Comment:
set this value to default, and doc the value to inform user to set the value properly